packages/storefront-lwc/src/modules/commerce/api/client.js

```diff
@@ -32,3 +33,5 @@ const apiClient = new ApolloClient({
 });
 
 export { apiClient };
+
+setClient(apiClient);
```

Comment:
I am just curious the order of these two steps matter?

Reply:
No, I don't think so